app/scrapers/yahoo.py

```diff
@@ -1,6 +1,6 @@
 from __future__ import print_function
 from .generalized import Scraper
-
+import re
```

Comment:
Why use __re.sub()__ when you can use [__'str'.replace()__](https://docs.python.org/3/library/stdtypes.html?#str.replace)?